camera/rpi-files/setup.sh

```diff
@@ -21,3 +21,7 @@ python3 -m venv env &&
 source env/bin/activate &&
 pip install -r requirements.txt &&
 deactivate
+
+# Download API generator
+wget https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/4.2.3/openapi-generator-cli-4.2.3.jar -O openapi-generator-cli.jar
```

Comment:
maybe add string var whith configurable version number

Reply:
even specified one version for the whole projects as we are also going to use it in kotlin app

Reply:
I would keep separate copy for embedded devices, we don't know how to ship them yet.

Reply:
But should it be like that every part of the system uses the same version of API generator java program to generate the needed classes etc.?

Reply:
Not necessarily, I think. API definition guarantees the contract, and clients just implement that.

Reply:
Not necessarily, I think. API definition guarantees the contract, and clients just implement that. E.g. camera doesn't care how Android implements the API and vice-versa.

Reply:
I moved generator version to a separate variable in this file, it's a good idea even if we don't share this value with other system parts.